grpc_unary_attestation/src/server.rs

```diff
@@ -140,12 +142,23 @@ where
     ) -> anyhow::Result<tonic::Response<UnaryResponse>, tonic::Status> {
         let error_logger = self.error_logger.clone();
         let request_inner = request.into_inner();
+        let session_id: SessionId = {
+            let boxed_slice: Box<SessionId> = request_inner
+                .session_id
+                .into_boxed_slice()
+                .try_into()
+                .map_err(|error| {
+                    error_logger.log_error(&format!("Received malformed session_id: {:?}", error));
+                    tonic::Status::invalid_argument("")
+                })?;
+            *boxed_slice
+        };
 
```

Comment:
Not super happy with this, I wish we could declare the array length in the proto instead, but protobuf does not support this. :/ Thoughts?

Reply:
I think this is as good as we can get with protos, but see my other comment re: returning an actionable error message to the client in case the precondition on the session id length is not met.